packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -789,7 +789,7 @@ function finishConcurrentRender(
         hasNotProcessedNewUpdates &&
         // do not delay if we're inside an act() scope
         !(
-          __DEV__ &&
+          (__DEV__ || __TESTING__) &&
```

Comment:
We need a plan for all the other warnings. Do we need to update all the other warning guards, too?

Reply:
I don’t think we have to change anything?